src/BloomBrowserUI/bookEdit/pageThumbnailList/pageControls/pageControls.less

```diff
@@ -1,5 +1,6 @@
 @import "../../../templates/common-mixins.less";
 
+@ButtonDarkerGray: #363333;
```

Comment:
Wasn't there a common color mixin file you'd made?

Reply:
That's a separate issue BL-5326.